### DISCUSSION TITLE
Nimib speaking hours 🎪 - Jan 29th, 10-11 CET - goals for 2023

### DISCUSSION THREAD

**Discussion by pietroppeter** [pietroppeter/nimib#162] [Nimib speaking hours]
hi @HugoGranstrom (and @Vindaar) and all those possibly interested!

**Confirmed Sunday (Jan 29th, 10-11 CET) link https://meet.jit.si/NimibSpeakingHours**

I was thinking of organizing a nimib speaking hours next Sunday (Jan 29th, 10-11 CET?). What do you think?
The plan would be to review what could be nimib priorities for nimib in 2023.

On top of my list are:
- move to nimib organization (and create nimibex repo)
- keep working on static site generator with nimib

but there are many smaller tasks, some of those are almost ready to be implemented.

**Comment by HugoGranstrom** [comment]
> next Sunday (Jan 29th, 10-11 CET?). What do you think?

Works for me :D 

> The plan would be to review what could be nimib priorities for nimib in 2023.

Sounds like a good plan, I'll read up on all the open issues. And we'll see how far I've gotten with the codeAsInSource as well.

**Comment by pietroppeter** [comment]
is there a way to publish some kind of calendar event that people can import in their google/whatnot calendars?

**Comment by pietroppeter** [comment]
we start in less than 15 minutes added the link

**Reply by pietroppeter** [reply]
I am actually going to start going in

**Comment by pietroppeter** [comment]
really enjoyed the first nimib speaking hours of the year, thanks @HugoGranstrom! I will add here later some notes about what we discussed

**Comment by pietroppeter** [comment]
notes from the meeting:
- the general theme was goals 2023
- we created the nimib organization (nimib name not available) https://github.com/nimib-org
- we create a board inside nimib-org and added a few tasks: https://github.com/orgs/nimib-org/projects/1 (just realized it was private, I made it public)
- update on Pietro's progress on building a nimib's SSG - goals is to have a release by first half of 2023
  - last month mostly spent on "side quests": yamly, jsony skipHook
  - next step: port https://github.com/dmulholl/ivy to nim as learning experience
  - maybe try also porting: https://www.webdevdrops.com/en/build-static-site-generator-nodejs-8969ebe34b22/
- ongoing work in nimibook
  - parallel/next PR merged
  - pietro: finish docs by feb and release 0.3
  - reviewed priorities for 2023
  - we created labels for priorities for stuff to be completed in first half ([2023H1](https://github.com/pietroppeter/nimibook/issues?q=is%3Aopen+is%3Aissue+label%3A2023H1)) and stuff to be completed in second half ([2023H2](https://github.com/pietroppeter/nimibook/issues?q=is%3Aopen+is%3Aissue+label%3A2023H2))
- we reviewed issues in nimib: created labels [2023H1](https://github.com/pietroppeter/nimib/labels/2023H1) and [2023H2](https://github.com/pietroppeter/nimib/labels/2023H2)
- we created nimibex repo and started creating issues on stuff we want to add
- brief overview on the rest of ecosystem
  - we will need to migrate nimislides, nimibook, nimiboost (maybe p5nim?)
  - nblog (make it multiauthor), feedo

note that all these plans on doing stuff come with a very loose open source commitment: we work on these projects on our free time, we do it because we enjoy it, we will **not** get stressed if we do not manage to complete our plans, whatever happens in life means we might need to redirect our energy elsewhere.

also note that you (the reader who arrived up to here) are most welcome to pick up any issues,  in principle even the ones we have assigned ourselves. in those case though please ask before putting any work on it. have a great week everyone! (yeah, that means just me and you @HugoGranstrom ;))

**Comment by pietroppeter** [comment]
btw something we forgot to discuss is date for next speaking hours and whether or not we plan to use some tool to publish it as an event that one can easy add to a calendar. Following a brief search and [options that were given on fosstodon](https://fosstodon.org/@pietroppeter/109766524790308477), I thought this https://joinmobilizon.org/ is interesting

**Reply by HugoGranstrom** [reply]
I gave it a quick look and it looks promising 👌

**Reply by HugoGranstrom** [reply]
As for the next date, I don't think I have much planned in general the coming months so I should be pretty flexible.

**Comment by pietroppeter** [comment]
actually probably it makes more sense to have milestones than labels for 2023H1. I have create one such milestone: https://github.com/pietroppeter/nimib/milestone/1
